CHANGELOG.asciidoc

```diff
@@ -31,6 +31,8 @@ https://github.com/elastic/beats/compare/v5.0.0-alpha5...master[Check the HEAD d
 ==== Bugfixes
 
 *Affecting all Beats*
+- Use indentation when generating the index pattern {pull}2217[2217]
```

Comment:
Do we even have to add it here as the script was not part of alpha5? Bugfix for something that was not release yet?